dhall/src/Dhall/Main.hs

```diff
@@ -225,6 +227,18 @@ parseMode =
                 <>  Options.Applicative.help "Read expression from a file instead of standard input"
                 <>  Options.Applicative.metavar "FILE"
                 )
+    
+    parseOutput = fmap f (optional p)
+      where
+        f Nothing = StandardOutput
+        f (Just file) = Output file
+
+        p = Options.Applicative.strOption
+                (   Options.Applicative.short 'o'
```

Comment:
So far `dhall` has largely refrained from using `short` options – I'm not sure why though. Probably we want the CLI commands to be as readable as possible too…

Thoughts, @Gabriel439?

Reply:
Okay, I'll just remove it.

Reply:
Yeah, the use of only long options is intentional.

In my experience, short options are an impediment to learning to beginners (especially if they learn from tutorial material that uses short options or by observing other people using short options).  Short options also tend to lead to people cargo-culting the options blindly without thinking about what they do.  This comment accurately summarizes what happens when short options become idiomatic:

https://xkcd.com/1168/